proc-macros/tests/ui/correct/custom_ret_types.rs

```diff
@@ -4,8 +4,7 @@ use std::net::SocketAddr;
 
 use jsonrpsee::core::{async_trait, ClientError, Serialize};
 use jsonrpsee::proc_macros::rpc;
-use jsonrpsee::server::{IntoResponse, ServerBuilder};
-use jsonrpsee::types::ResponsePayload;
+use jsonrpsee::server::{IntoResponse, ResponsePayload, ServerBuilder};
```

Comment:
This is a breaking change as well

Reply:
Aah I see; it moved! So will you do a proper release then instead of a patch one do you think? Or alias this type in `jsonrpsee::types` in the release to avoid this break?

Reply:
I think we have to make a proper release because the MethodResponse is also a breaking change and I can't do proper alias in types because then the `feature flag core/server` needs to be enabled for ResponsePayload to be available.